docker-compose/build.sh

```diff
@@ -2,5 +2,7 @@
 set -e
 ( git clone https://github.com/ewasm/ewasm-studio && cd ewasm-studio && docker build -t ewasm/ewasm-studio . )
```

Comment:
I think this script should take a tag as an input argument (if not supplied, take `master`).

Reply:
it can now take repo/branch as parameters for hera, geth, ewasm studio and etherchain light.